app/forms/nomination_form.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+class NominationForm
+  include ActiveModel::Model
+  include ActiveModel::Serialization
+
+  attr_accessor :local_authority_id, :school_id
+  validates :local_authority_id,
+            presence: { message: "We could not find any local authorities matching your search criteria" },
+            on: %i[local_authority save]
+  validates :school_id,
+            presence: { message: "We could not find any establishments matching your search criteria" },
+            on: %i[school save]
+
+  def attributes
```

Comment:
Where are we using this?

Reply:
```
include ActiveModel::Serialization
```
requires it. 

And we use that in `serializable_hash` in the controller.